splunk_otel/cmd/trace.py

```diff
@@ -34,7 +36,15 @@
     required=False,
     type=str,
     dest="token",
-    help="Your SignalFx Access Token (SIGNALFX_ACCESS_TOKEN env var by default)",
+    help="Your Splunk Access Token (SPLUNK_ACCESS_TOKEN env var by default)",
```

Comment:
Side comment not directly related, but probably worth having a short discussion.

Were the options of passing the access token reviewed from a security perspective?

The safest way I am aware of is to allow the user to pass the password via STDIN. I think it would be good to allow such a possibility. Sill, I think that we should keep the options to get it via env vars and CLI arg (for sake of ease of use). But it could be also good to tell the users that using STDIN to get sensitive data (if implemented correctly) is more secure.

Reply:
I ported most features including this one from SignalFx libraries. So any security review those libs had before becoming part of Splunk would apply. 

STDIN sounds more useful for CLI tools that humans invoke manually. This will most likely be used by upstart/systemd like process managers and automated so not sure how useful STDIN would be here.